build.gradle

```diff
@@ -35,7 +35,7 @@ apply plugin: 'io.codearte.nexus-staging'
 
 allprojects {
     group = 'io.minio'
-    version = '6.0.12'
+    version = '7.0.0'
```

Comment:
major version change should remove all deprecated methods

Reply:
@balamurugana  ptal